content/courses/program-optimization/rust-macros.md

```diff
@@ -451,12 +454,12 @@ Finally, the `expanded` variable is converted into a `TokenStream` using the
 `into()` method.
 
 ```rust
-use proc_macro::{self, TokenStream};
+use proc_macro::TokenStream;
 use quote::quote;
-use syn::{parse_macro_input, DeriveInput, FieldsNamed};
+use syn::{parse_macro_input, DeriveInput, Data, Fields};
 
 #[proc_macro_derive(Describe)]
-pub fn describe(input: TokenStream) -> TokenStream {
+pub fn describe_struct(input: TokenStream) -> TokenStream {
```

Comment:
This method needs rework in order to be inline with your previous changes (import usage, usage of quote! inside format!, etc). Looks like you forgot to update this section according to your previous changes

Also, text is saying that a describe() method will be implemented, and you're implementing a describe_struct() method. Please revise methods naming